uninstall.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Fired when the plugin is uninstalled.
+ */
+ 
+/* If uninstall not called from WordPress, then exit. */
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	wp_die( sprintf( __( '%s should only be called when uninstalling the plugin.', 'wdp-plugin' ), '<code>' . __FILE__ . '</code>' ) );
+}
+
+
+/* Deleting FFPW widget option row in database */
+delete_option('widget_FFPW_Flexible_Featured_Post');
+
+
+/* Deleting post meta data added by plugin */
+function delete_ffpw_meta() {
+   
+    global $wpdb;
```

Comment:
Good question -  since it isn’t actually needed to delete the post meta, there doesn’t seem to be a point in keeping it. In the wp-includes/post.php file delete_post_meta calls delete_metadata, which uses the global $wpdb variable, but it is defined in that file. So I will remove it here.